engines/scumm/dimuse_v1/dimuse_bndmgr.cpp

```diff
@@ -25,8 +25,8 @@
 #include "scumm/scumm.h"
 #include "scumm/util.h"
 #include "scumm/file.h"
-#include "scumm/imuse_digi/dimuse_bndmgr.h"
-#include "scumm/imuse_digi/dimuse_codecs.h"
+#include "scumm/dimuse_v1/dimuse_bndmgr.h"
```

Comment:
actually don't rename name. keep as original name. for v2 use dimuse

Reply:
I am a bit confused, are we using: `IMuseDigital` (abstract class), `IMuseDigitalV1` and `IMuseDigitalV2`?
Or `GenericIMuseDigital` (abstract class), `IMuseDigital` and `DiMUSE`?

Reply:
hmm, sorry for confusion. Now, I'm thinking about to not have two separated on same level. Maybe abstract one change name class to IMuseDigitalAbstract in new file which in future will disappear once v1 will not be needed anymore.
New/V2 named it IMuseDigital as target name. old one named it to IMuseDigitalV1. keep same folder. do not rename files. I want avoid moving files and renames. for v2 file name clash you can add some new name dimuse_core.cpp maybe?

Reply:
No problem, thanks for the clarification; the reason why the V1/V2 naming made sense to me is because even if CMI and DIG code disappears in V1 in the future, V1 still has to handle FT (which effectively is an earlier but very different version of the engine even in the disasm), so semantically it makes sense.

This is also why I'm in favor of putting "_core" suffix in every v2 filename: I want to avoid confusion between modules of the FT engine and modules of CMI/DIG engine.

Reply:
FT imuse is earlier version, but simpler. it can be integrated into into v2. similar like in v1. so no two versions. FT is not that different. there is only one imuse.

Reply:
Okay, sounds good, I will do like you said